src/eddington/fit_data.py

```diff
@@ -367,3 +368,20 @@ def __is_number(cls, string):
             return True
         except ValueError:
             return False
+
+    def set_cell(self, row, col, value):
+        """
+        Allowes to set value to a cell.
+
+        :param row: The row number, where 0 represents the header row
+        :param col: The columns name (original, so that it can be changed here)
+        :param value: The new value to set for the cell
+        """
+        if row != 0:
```

Comment:
You should split this function to 2 seperated function:
1. `set_header` which allow you to rename a header. This can be done using `pop` and assign as in [here](https://stackoverflow.com/a/16475444/6624798)
2. `set_cell` that allows you to set a data cell, where the count starts from 1 and.

Reply:
For OrderedDict, the answer you referred me to does almost exactly what is done for headers in here.
Did you mean something else?